pkg/admin/prerun/prerun.go

```diff
@@ -67,17 +79,26 @@ func (dm *DataManager) Run() error {
 		[0] it would need a comprehensive check if data exists, not just existence of /var/lib/microshift
 	*/
 
+	if dataExists && versionExists && healthExists {
+		// 8 - regular flow
+		return false, nil
+	}
+
+	klog.InfoS("Handling special cases of pre-run data management")
+	defer klog.InfoS("Completed handling special cases of pre-run data management")
```

Comment:
Should we move these to the beginning of the function? Otherwise, we may get errors without the context, right?